### PR TITLE
feat positive tests for api

### PR DESCRIPTION
**Pull request by BPylypenko**

### CODE REVIEW

File: packages/cli/odahuflow/cli/parsers/route.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def create(client: ModelRouteClient, mr_id: str, file: str, wait: bool, timeout:
     Create a model route.\n
     You should specify a path to file with a model route. The file must contain only one model route.
     For now, CLI supports yaml and JSON file formats.
-    If you want to create multiples routes than you should use "odahuflowctl res apply" instead.
+    If you want to create multiples routes than you should use "odahuflowctl bulk apply" instead.
```
Comment (keshamin) [inline packages/cli/odahuflow/cli/parsers/route.py:85]:
```suggestion
    If you want to create multiples routes then you should use "odahuflowctl bulk apply" instead.
```
The same can be applied to all 3 changes in this file.

File: packages/tests/e2e/robot/tests/api/resources/packager/valid/docker_cli_create.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      properties:
+        - name: dockerfileBaseImage
+          parameters:
+            - name: description
+              value: Base image for Dockerfile.
+            - name: type
+              value: string
+            - name: default
+              value: >-
+                gcr.io/or2-msq-epmd-legn-t1iylu/odahu/odahu-flow-docker-packager-base:1.2.0-b1592953604635
```
Comment (karbyshevds) [inline packages/tests/e2e/robot/tests/api/resources/packager/valid/docker_cli_create.yaml:31]: Is it safe to use dev build number? Maybe we can pass this as argument or use some permanent tag?

File: packages/tests/e2e/robot/tests/api/resources/packager/valid/docker_cli_update.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                "name": "description",
+                "value": "Base image for Dockerfile."
+              },
+              {
+                "name": "type",
+                "value": "string"
+              },
+              {
+                "name": "default",
+                "value": "gcr.io/or2-msq-epmd-legn-t1iylu/odahu/odahu-flow-docker-packager-base:1.2.0-b1592953604635"
```
Comment (karbyshevds) [inline packages/tests/e2e/robot/tests/api/resources/packager/valid/docker_cli_update.json:44]: Is it safe to use dev build number? Maybe we can pass this as argument or use some permanent tag?

File: packages/tests/e2e/robot/tests/api/resources/packager/valid/docker_rest_create.json [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+                "name": "description",
+                "value": "Base image for Dockerfile."
+              },
+              {
+                "name": "type",
+                "value": "string"
+              },
+              {
+                "name": "default",
+                "value": "gcr.io/or2-msq-epmd-legn-t1iylu/odahu/odahu-flow-docker-packager-base:1.2.0-b1592953604635"
```
Comment (karbyshevds) [inline packages/tests/e2e/robot/tests/api/resources/packager/valid/docker_rest_create.json:44]: Is it safe to use dev build number? Maybe we can pass this as argument or use some permanent tag?

File: packages/tests/e2e/robot/tests/api/resources/packager/valid/docker_rest_update.yaml [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+      properties:
+        - name: dockerfileBaseImage
+          parameters:
+            - name: description
+              value: Base image for Dockerfile.
+            - name: type
+              value: string
+            - name: default
+              value: >-
+                gcr.io/or2-msq-epmd-legn-t1iylu/odahu/odahu-flow-docker-packager-base:1.2.0-b1592953604635
```
Comment (karbyshevds) [inline packages/tests/e2e/robot/tests/api/resources/packager/valid/docker_rest_update.yaml:31]: Is it safe to use dev build number? Maybe we can pass this as argument or use some permanent tag?

File: packages/tests/e2e/robot/tests/api/resources/packager/valid/docker_cli_create.yaml (hunk 2 above)
Comment (BPylypenko) [reply packages/tests/e2e/robot/tests/api/resources/packager/valid/docker_cli_create.yaml:31]: I will just delete these values as they are not needed, at least, for now in testing.

File: packages/tests/e2e/robot/tests/api/API_training_packaging_deploy_route_model.robot [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+
+Get info about model
+    [Tags]                      model
+    ${result}                   Call API  model get  url=${MODEL_URL}
+    should be equal             ${result['info']['description']}  This is a EDI server.
+
+Invoke model
+    [Tags]                      model
+    ${result}                   Call API  model post  url=${MODEL_URL}  json_input=${REQUEST}
+    should be equal as strings  ${result}  {'prediction': [4.675934265196686], 'columns': ['quality']}
```
[the remark this answers is not in the document]
Comment (BPylypenko) [reply packages/tests/e2e/robot/tests/api/API_training_packaging_deploy_route_model.robot:173]: OK, I agree. As I checked, the `${result}` is `<class 'dict'>`. So, I have converted the expected response to dict and checked the equality of dictionaries.
Is it a feasible solution?

**Pull request merged**